lib/release_notes/github_api.rb

```diff
@@ -35,17 +34,32 @@ def merged_pull_requests(old_sha)
     end
 
     def closed_pull_requests_between(old_sha)
-      closed_pull_requests.take_while do |pr|
-        pr.updated_at > last_commit_date(old_sha)
+      end_date = last_commit_date(old_sha)
+      page ||= 1
+      closed_prs = []
+
+      loop do
+        current_prs = closed_pull_requests(page: page)
+        prs = current_prs.take_while do |pr|
+          pr.updated_at > end_date
+        end
+
+        closed_prs << prs
+        page += 1
+
+        break unless prs.count == current_prs.count
       end
+
+      return closed_prs.flatten
     end
 
     def last_commit_date(old_sha)
```

Comment:
Any reason we can just call this `commit_date`? There is nothing about the method that makes it not usable for retrieving any commit date as far as I can see.